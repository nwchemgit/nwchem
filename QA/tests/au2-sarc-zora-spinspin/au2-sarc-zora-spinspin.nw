echo
start au2-sarc-zora-spinspin

geometry
Au 0.000 0.000 -1.25
Au 0.000 0.000  1.25
end

basis spherical
* library sarc-zora
end


dft
xc pbe0
convergence  energy 1d-10
noprint "final vectors analysis" multipole
grid fine
end

relativistic
 zora on
 zora:cutoff 1d-30
 modelpotential 1
end

property
shielding
spinspin
end

set int:cando_txs f

task dft property
