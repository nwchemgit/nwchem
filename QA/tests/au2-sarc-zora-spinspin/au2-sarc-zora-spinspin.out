 argument  1 = /Users/edo/nwchem/nwchem/QA/tests/au2-sarc-zora-spinspin/au2-sarc-zora-spinspin.nw
  NWChem w/ OpenMP: maximum threads =    1



============================== echo of input deck ==============================
echo
start au2-sarc-zora-spinspin

geometry
Au 0.000 0.000 -1.25
Au 0.000 0.000  1.25
end

basis spherical
* library sarc-zora
end


dft
xc pbe0
convergence  energy 1d-10
noprint "final vectors analysis" multipole
grid fine
end

relativistic
 zora on
 zora:cutoff 1d-30
 modelpotential 1
end

property
shielding
spinspin
end

set int:cando_txs f

task dft property
================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 7.0.1
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2020
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = WE40672
    program         = /Users/edo/nwchem/nwchem/bin/MACX64/nwchem
    date            = Thu Feb 17 13:05:30 2022

    compiled        = Thu_Feb_17_10:02:54_2022
    source          = /Users/edo/nwchem/nwchem
    nwchem branch   = 7.0.0
    nwchem revision = nwchem_on_git-3195-g27fae8c638
    ga revision     = 5.8.1
    use scalapack   = T
    input           = /Users/edo/nwchem/nwchem/QA/tests/au2-sarc-zora-spinspin/au2-sarc-zora-spinspin.nw
    prefix          = au2-sarc-zora-spinspin.
    data base       = ./au2-sarc-zora-spinspin.db
    status          = startup
    nproc           =        2
    time left       =     -1s



           Memory information
           ------------------

    heap     =   52195779 doubles =    398.2 Mbytes
    stack    =   52195783 doubles =    398.2 Mbytes
    global   =   29826161 doubles =    227.6 Mbytes (distinct from heap & stack)
    total    =  134217723 doubles =   1024.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 ORDER OF PRIMARY AXIS IS BEING SET TO 4
 D4H symmetry detected

          ------
          auto-z
          ------
  no constraints, skipping    0.0000000000000000     
  no constraints, skipping    0.0000000000000000     


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 Au                  79.0000     0.00000000     0.00000000    -1.25000000
    2 Au                  79.0000     0.00000000     0.00000000     1.25000000

      Atomic Mass 
      ----------- 

      Au               196.966600


 Effective nuclear repulsion energy (a.u.)    1321.0380844028

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             D4h       
 Group number             28
 Group order              16
 No. of unique centers     1

      Symmetry unique atoms

     1



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       2.50000


            XYZ format geometry
            -------------------
     2
 geometry
 Au                    0.00000000     0.00000000    -1.25000000
 Au                    0.00000000     0.00000000     1.25000000

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 Au               |   1 Au               |     4.72431  |     2.50000
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          1
 ==============================================================================





 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                         sarc-zora                  on all atoms 


                              NWChem Property Module
                              ----------------------


  itol2e modified to match energy
  convergence criterion.

                                 NWChem DFT Module
                                 -----------------


                      Basis "ao basis" -> "ao basis" (spherical)
                      -----
  Au (Gold)
  ---------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.73343595E+06  0.059081
  1 S  7.70415977E+05 -0.013465
  1 S  3.42407101E+05  0.126692
  1 S  1.52180934E+05  0.091737
  1 S  6.76359705E+04  0.330193
  1 S  3.00604313E+04  0.534847

  2 S  1.33601917E+04  1.000000

  3 S  5.93786298E+03  1.000000

  4 S  2.63905021E+03  1.000000

  5 S  1.17291121E+03  1.000000

  6 S  5.21293869E+02  1.000000

  7 S  2.31686164E+02  1.000000

  8 S  1.02971629E+02  1.000000

  9 S  4.57651680E+01  1.000000

 10 S  2.03400750E+01  1.000000

 11 S  9.04003300E+00  1.000000

 12 S  4.01779300E+00  1.000000

 13 S  1.78568600E+00  1.000000

 14 S  7.93638000E-01  1.000000

 15 S  3.52728000E-01  1.000000

 16 S  1.56768000E-01  1.000000

 17 S  6.96750000E-02  1.000000

 18 P  2.42948435E+04  0.011587
 18 P  9.71793740E+03  0.017254
 18 P  3.88717496E+03  0.081452
 18 P  1.55486998E+03  0.253008
 18 P  6.21947994E+02  0.739309

 19 P  2.48779197E+02  1.000000

 20 P  9.95116790E+01  1.000000

 21 P  3.98046720E+01  1.000000

 22 P  1.59218690E+01  1.000000

 23 P  6.36874700E+00  1.000000

 24 P  2.54749900E+00  1.000000

 25 P  1.01900000E+00  1.000000

 26 P  4.07600000E-01  1.000000

 27 P  1.63040000E-01  1.000000

 28 P  6.52160000E-02  1.000000

 29 D  1.86912758E+03  0.007709
 29 D  6.79682755E+02  0.043860
 29 D  2.47157366E+02  0.245339
 29 D  8.98754060E+01  0.806013

 30 D  3.26819660E+01  1.000000

 31 D  1.18843510E+01  1.000000

 32 D  4.32158200E+00  1.000000

 33 D  1.57148400E+00  1.000000

 34 D  5.71449000E-01  1.000000

 35 D  2.07800000E-01  1.000000

 36 D  7.55630000E-02  1.000000

 37 F  9.08433260E+01  0.060535
 37 F  3.02811090E+01  0.290111
 37 F  1.00937030E+01  0.569028
 37 F  3.36456800E+00  0.377737

 38 F  1.12152300E+00  1.000000

 39 F  3.73841000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 Au                        sarc-zora                39      111   17s11p8d3f




 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 Au                        sarc-zora                39      111   17s11p8d3f



 Using Relativistic ZORA Model Potential Approach
 C. van Wullen, J. Chem. Phys. 109, 392 (1998)
 C. van Wullen, C. Michauk, J. Chem. Phys. 123, 204113 (2005)

 Reading Model Potential File:
 /Users/edo/nwchem/nwchem/src/basis/libraries/rel-modelpotentials/modbas.4c

  int_init: cando_txs set to always be  F
  Caching 1-el integrals 
  tol_rho modified to match energy
  convergence criterion.

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     2
          No. of electrons :   158
           Alpha electrons :    79
            Beta electrons :    79
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  50
          AO basis - number of functions:   222
                     number of shells:    78
          Convergence on energy requested:  1.00D-10
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                         PBE0 Method XC Functional
                     Hartree-Fock (Exact) Exchange  0.250          
          PerdewBurkeErnzerhof Exchange Functional  0.750          
            Perdew 1991 LDA Correlation Functional  1.000 local    
           PerdewBurkeErnz. Correlation Functional  1.000 non-local

             Grid Information
             ----------------
          Grid used for XC integration:  fine      
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          Au                  1.35      141          11.0       974
          Grid pruning is: on 
          Number of quadrature shells:   282
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         50 iters            50 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-13
          AO Gaussian exp screening on grid/accAOfunc:  23
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-11

          Performing ZORA calculations
          ----------------------------


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:      -35687.04229344

 Read atomic ZORA corrections from ./au2-sarc-zora-spinspin.zora_sf

 dft_zora_read: failed to open./au2-sarc-zora-spinspin.zora_sf
       Generating atomic ZORA corrections
       ----------------------------------

 In dft_scf:: zora:Knucl= F

 Grid_pts file          = ./au2-sarc-zora-spinspin.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =     51        Max. recs in file   = 506625431


 Wrote atomic ZORA corrections to ./au2-sarc-zora-spinspin.zora_sf


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:      -35687.04229344

      Non-variational initial energy
      ------------------------------

 Total energy =  -35687.058137
 1-e energy   =  -51935.569939
 2-e energy   =   14927.473717
 HOMO         =      -0.396397
 LUMO         =      -0.125365

   Time after variat. SCF:     28.4
   Time prior to 1st pass:     28.4
 Grid integrated density:     158.000022752012
 Requested integration accuracy:   0.10E-06

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       18.01            18013251
          Stack Space remaining (MW):       52.10            52095940

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1 -38152.0825714718 -3.95D+04  5.28D-02  9.94D+07    60.7
 Grid integrated density:     158.000020427053
 Requested integration accuracy:   0.10E-06
 d= 0,ls=0.0,diis     2 -39313.2492649941 -1.16D+03  1.49D-02  3.90D+03    94.3
 Grid integrated density:     158.000017576392
 Requested integration accuracy:   0.10E-06
 d= 0,ls=0.0,diis     3 -39314.4381583097 -1.19D+00  1.89D-02  2.16D+02   124.4
 Grid integrated density:     158.000022449918
 Requested integration accuracy:   0.10E-06
 d= 0,ls=0.0,diis     4 -39314.7833735142 -3.45D-01  1.29D-02  7.28D+01   156.9
 Grid integrated density:     158.000019258110
 Requested integration accuracy:   0.10E-06
 d= 0,ls=0.0,diis     5 -39316.4343852110 -1.65D+00  3.72D-03  2.43D+00   190.3
 Grid integrated density:     158.000020095446
 Requested integration accuracy:   0.10E-06
 d= 0,ls=0.0,diis     6 -39316.5348356382 -1.00D-01  1.26D-03  2.04D-01   222.1
 Grid integrated density:     158.000019989461
 Requested integration accuracy:   0.10E-06
  Resetting Diis
 d= 0,ls=0.0,diis     7 -39316.5483507473 -1.35D-02  2.62D-04  1.65D-02   253.2
 d= 0,ls=0.0,diis     8 -39316.5494694799 -1.12D-03  1.02D-04  2.82D-04   283.1
 d= 0,ls=0.0,diis     9 -39316.5494563477  1.31D-05  4.43D-05  8.68D-04   312.6
 d= 0,ls=0.0,diis    10 -39316.5494980467 -4.17D-05  1.13D-05  1.47D-05   342.0
 d= 0,ls=0.0,diis    11 -39316.5494992236 -1.18D-06  7.33D-06  9.34D-07   371.5
 d= 0,ls=0.0,diis    12 -39316.5494993714 -1.48D-07  4.01D-06  3.85D-07   400.8
 d= 0,ls=0.0,diis    13 -39316.5494994271 -5.56D-08  1.07D-06  1.23D-08   430.3
 d= 0,ls=0.0,diis    14 -39316.5494994295 -2.40D-09  3.28D-08  1.87D-11   459.8
 d= 0,ls=0.0,diis    15 -39316.5494994293  1.82D-10  1.18D-08  1.53D-11   489.9
 d= 0,ls=0.0,diis    16 -39316.5494994292  5.09D-11  2.06D-09  1.88D-13   520.5

      Commencing ZORA Property Calculations
      -------------------------------------

dia1(k,t,iat1,ispin)=(  1,  1,  1,  1)=(  9417.70838374,  9417.70838374) ppm
dia1(k,t,iat1,ispin)=(  1,  2,  1,  1)=(     0.00000000,     0.00000000) ppm
dia1(k,t,iat1,ispin)=(  1,  3,  1,  1)=(    -0.00000000,    -0.00000000) ppm
dia1(k,t,iat1,ispin)=(  2,  1,  1,  1)=(     0.00000000,     0.00000000) ppm
dia1(k,t,iat1,ispin)=(  2,  2,  1,  1)=(  9417.70838374,  9417.70838374) ppm
dia1(k,t,iat1,ispin)=(  2,  3,  1,  1)=(     0.00000000,     0.00000000) ppm
dia1(k,t,iat1,ispin)=(  3,  1,  1,  1)=(     0.00000000,     0.00000000) ppm
dia1(k,t,iat1,ispin)=(  3,  2,  1,  1)=(    -0.00000000,    -0.00000000) ppm
dia1(k,t,iat1,ispin)=(  3,  3,  1,  1)=(  9432.92856223,  9432.92856223) ppm
(dia2,par1)(k,t,iat1,spin)=(  1,  1,  1,  1)=(     3.70290371,     3.70290371,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  2,  1,  1,  1)=(    -0.00000000,    -0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  3,  1,  1,  1)=(     0.00000000,     0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  1,  2,  1,  1)=(    -0.00000000,    -0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  2,  2,  1,  1)=(     3.70290371,     3.70290371,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  3,  2,  1,  1)=(     0.00000000,     0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  1,  3,  1,  1)=(    -0.00000000,    -0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  2,  3,  1,  1)=(     0.00000000,     0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  3,  3,  1,  1)=(     0.00000000,     0.00000000,     0.00000000,     0.00000000) ppm
dia1(k,t,iat1,ispin)=(  1,  1,  2,  1)=(  9417.70838368,  9417.70838368) ppm
dia1(k,t,iat1,ispin)=(  1,  2,  2,  1)=(     0.00000000,     0.00000000) ppm
dia1(k,t,iat1,ispin)=(  1,  3,  2,  1)=(    -0.00000000,    -0.00000000) ppm
dia1(k,t,iat1,ispin)=(  2,  1,  2,  1)=(     0.00000000,     0.00000000) ppm
dia1(k,t,iat1,ispin)=(  2,  2,  2,  1)=(  9417.70838368,  9417.70838368) ppm
dia1(k,t,iat1,ispin)=(  2,  3,  2,  1)=(     0.00000000,     0.00000000) ppm
dia1(k,t,iat1,ispin)=(  3,  1,  2,  1)=(     0.00000001,     0.00000001) ppm
dia1(k,t,iat1,ispin)=(  3,  2,  2,  1)=(    -0.00000000,    -0.00000000) ppm
dia1(k,t,iat1,ispin)=(  3,  3,  2,  1)=(  9432.92856212,  9432.92856212) ppm
(dia2,par1)(k,t,iat1,spin)=(  1,  1,  2,  1)=(     3.70290370,     3.70290370,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  2,  1,  2,  1)=(    -0.00000000,    -0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  3,  1,  2,  1)=(     0.00000000,     0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  1,  2,  2,  1)=(    -0.00000000,    -0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  2,  2,  2,  1)=(     3.70290370,     3.70290370,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  3,  2,  2,  1)=(     0.00000000,     0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  1,  3,  2,  1)=(     0.00000000,     0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  2,  3,  2,  1)=(    -0.00000000,    -0.00000000,     0.00000000,     0.00000000) ppm
(dia2,par1)(k,t,iat1,spin)=(  3,  3,  2,  1)=(     0.00000000,     0.00000000,     0.00000000,     0.00000000) ppm

 Wrote ZORA NMR data to ./au2-sarc-zora-spinspin.zora_nmrcs

(nogshift,skip_gshiftAOev,done_Fji)=( 1,F,T)


         Total DFT energy =   -39316.549499429231
      One electron energy =   -56031.887569042665
           Coulomb energy =    16129.094424949206
    Exchange-Corr. energy =     -734.794439738619
 Nuclear repulsion energy =     1321.038084402847

       Scaling correction =     1343.554765742238

 Numeric. integr. density =      157.999999999169

     Total iterative time =    726.9s



 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
        2198.063737184906           0.000000000000           0.000000000000
           0.000000000000        2198.063737184906           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

      -----------------------------------
      Indirect Spin-Spin Coupling Tensors
      -----------------------------------


      Indirect spin-spin coupling not calculated for RHF
      wave functions: needs UHF wave function

            Scalar ZORA NMR Shielding
            -------------------------


          -----------------------------------------
          Chemical Shielding Tensors (GIAO, in ppm)
          -----------------------------------------

  int_init: cando_txs set to always be  F

 Read ZORA NMR data from ./au2-sarc-zora-spinspin.zora_nmrcs

                                NWChem CPHF Module
                                ------------------


  int_init: cando_txs set to always be  F
  scftype          =     RHF 
  nclosed          =       79
  nopen            =        0
  variables        =    11297
  # of vectors     =        3
  tolerance        = 0.10D-03
  level shift      = 0.00D+00
  max iterations   =       50
  max subspace     =       30

 SCF residual:    2.8541010972886145E-008


Iterative solution of linear equations
  No. of variables    11297
  No. of equations        3
  Maximum subspace       30
        Iterations       50
       Convergence  1.0D-04
        Start time   1107.7


   iter   nsub   residual    time
   ----  ------  --------  ---------
     1      3    1.60D-01    1179.1
     2      6    2.48D-02    1247.5
     3      9    1.16D-03    1316.0
     4     12    1.30D-04    1385.5
     5     15    8.85D-06    1457.2

 Wrote CPHF data to ./au2-sarc-zora-spinspin.shieldcphf

  int_init: cando_txs set to always be  F
 Calc. par tensor-> zora
      Atom:    1  Au
        Diamagnetic
   9421.4113     -0.0000     -0.0000
     -0.0000   9421.4113      0.0000
     -0.0000      0.0000   9432.9286

        Paramagnetic
   -628.3694     -0.0000      0.0000
     -0.0000   -628.3694     -0.0000
      0.0000     -0.0000    296.0678

        Total Shielding Tensor
   8793.0419     -0.0000      0.0000
     -0.0000   8793.0419     -0.0000
      0.0000     -0.0000   9728.9963

           isotropic =    9105.0267
          anisotropy =     935.9545

          Principal Components and Axis System
                 1           2           3
             9728.9963   8793.0419   8793.0419

      1         0.0000      0.7608      0.6489
      2        -0.0000     -0.6489      0.7608
      3         1.0000     -0.0000     -0.0000



      Atom:    2  Au
        Diamagnetic
   9421.4113     -0.0000      0.0000
     -0.0000   9421.4113     -0.0000
      0.0000     -0.0000   9432.9286

        Paramagnetic
   -628.3694      0.0000      0.0000
      0.0000   -628.3694     -0.0000
      0.0000     -0.0000    296.0678

        Total Shielding Tensor
   8793.0419      0.0000      0.0000
      0.0000   8793.0419     -0.0000
      0.0000     -0.0000   9728.9963

           isotropic =    9105.0267
          anisotropy =     935.9545

          Principal Components and Axis System
                 1           2           3
             9728.9963   8793.0419   8793.0419

      1         0.0000      0.6752      0.7376
      2        -0.0000      0.7376     -0.6752
      3         1.0000     -0.0000     -0.0000




 Task  times  cpu:     1457.3s     wall:     1457.3s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays


MA_summarize_allocated_blocks: starting scan ...
heap block 'gridpts', handle 103, address 0x11aa22f18:
	type of elements:		double precision
	number of elements:		33554432
	address of client space:	0x11aa22f80
	index for client space:		17102897
	total number of bytes:		268435568
MA_summarize_allocated_blocks: scan completed: 1 heap block, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         1	         0
	maximum number of blocks	        60	        60
	current total bytes		 268435568	         0
	maximum total bytes		 288053672	  43386840
	maximum total K-bytes		    288054	     43387
	maximum total M-bytes		       289	        44


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, Y. Alexeev, J. Anchell,
       V. Anisimov, F. W. Aquino, R. Atta-Fynn, J. Autschbach, N. P. Bauman,
     J. C. Becca, D. E. Bernholdt, K. Bhaskaran-Nair, S. Bogatko, P. Borowski,
         J. Boschen, J. Brabec, A. Bruner, E. Cauet, Y. Chen, G. N. Chuev,
      C. J. Cramer, J. Daily, M. J. O. Deegan, T. H. Dunning Jr., M. Dupuis,
   K. G. Dyall, G. I. Fann, S. A. Fischer, A. Fonari, H. Fruchtl, L. Gagliardi,
      J. Garza, N. Gawande, S. Ghosh, K. Glaesemann, A. W. Gotz, J. Hammond,
       V. Helms, E. D. Hermes, K. Hirao, S. Hirata, M. Jacquelin, L. Jensen,
   B. G. Johnson, H. Jonsson, R. A. Kendall, M. Klemm, R. Kobayashi, V. Konkov,
      S. Krishnamoorthy, M. Krishnan, Z. Lin, R. D. Lins, R. J. Littlefield,
      A. J. Logsdail, K. Lopata, W. Ma, A. V. Marenich, J. Martin del Campo,
   D. Mejia-Rodriguez, J. E. Moore, J. M. Mullin, T. Nakajima, D. R. Nascimento,
    J. A. Nichols, P. J. Nichols, J. Nieplocha, A. Otero-de-la-Roza, B. Palmer,
    A. Panyala, T. Pirojsirikul, B. Peng, R. Peverati, J. Pittner, L. Pollack,
   R. M. Richard, P. Sadayappan, G. C. Schatz, W. A. Shelton, D. W. Silverstein,
   D. M. A. Smith, T. A. Soares, D. Song, M. Swart, H. L. Taylor, G. S. Thomas,
            V. Tipparaju, D. G. Truhlar, K. Tsemekhman, T. Van Voorhis,
      A. Vazquez-Mayagoitia, P. Verma, O. Villa, A. Vishnu, K. D. Vogiatzis,
        D. Wang, J. H. Weare, M. J. Williamson, T. L. Windus, K. Wolinski,
        A. T. Wong, Q. Wu, C. Yang, Q. Yu, M. Zacharias, Z. Zhang, Y. Zhao,
                                and R. J. Harrison
                        "NWChem: Past, present, and future
                         J. Chem. Phys. 152, 184102 (2020)
                               doi:10.1063/5.0004997

                                      AUTHORS
                                      -------
     E. Apra, E. J. Bylaska, N. Govind, K. Kowalski, M. Valiev, W. A. de Jong,
      T. P. Straatsma, H. J. J. van Dam, D. Wang, T. L. Windus, N. P. Bauman,
       A. Panyala, J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec,
    K. Lopata, S. A. Fischer, S. Krishnamoorthy, M. Jacquelin, W. Ma, M. Klemm,
       O. Villa, Y. Chen, V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler,
           Eric Hermes, L. Jensen, J. E. Moore, J. C. Becca, V. Konjkov,
            D. Mejia-Rodriguez, T. Risthaus, M. Malagoli, A. Marenich,
   A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao,
        P.-D. Fan, A. Fonari, M. J. Williamson, R. J. Harrison, J. R. Rehr,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
      M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, M. Swart, Q. Wu,
   T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros,
     G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols,
       K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski,
       T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood,
      E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju,
        R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:     1457.5s     wall:     1457.5s
